release notes/v0.7.0.md

```diff
@@ -0,0 +1,51 @@
+xk6-browser v0.7.0 is here! :tada:
+
+We've been busy preparing for big changes in the future, so this minor release is smaller than usual. It contains a new feature to unblock users using `dialog` boxes, and internal improvements.
+
+
+## New features
+
+- `xk6-browser` will now automatically dismiss dialog boxes (`alert`, `confirm`, `prompt`, and `beforeunload`). ([#663](https://github.com/grafana/xk6-browser/pull/663))
+
+
+## Improvements
+
+- You can now launch a new browser without defining an empty options object. ([#649](https://github.com/grafana/xk6-browser/pull/649))
+    
+    With the empty object:
+
+    ```js
+    const browser = chromium.launch({});
+    ```
+
+    Without the empty object:
+
+    ```js
+    const browser = chromium.launch();
+    ```
+
+
+### Documentation
+
+- Updated the video to match what we released in `v0.6.0`, and it's still relevant for `v0.7.0`. ([#639](https://github.com/grafana/xk6-browser/pull/639))
+
+- Simplified the README. You will now be redirected to the [documentation](https://k6.io/docs/javascript-api/xk6-browser/). ([#656](https://github.com/grafana/xk6-browser/pull/656))
+
+- The project roadmap has been updated. ([#658](https://github.com/grafana/xk6-browser/pull/658))
+
+
+## Internals
+
+- Upgraded `k6` dependency to `v0.42.0`. ([#?](https://github.com/grafana/xk6-browser/pull/?))
+
+- Refactored the code that handles the lifecycle events. This shouldn't change any of the existing behavior, but if you notice anything that doesn't work as you expected please let us know. ([#647](https://github.com/grafana/xk6-browser/pull/647), [#644](https://github.com/grafana/xk6-browser/pull/644))
```

Comment:
Added the explanation below and "actionized" what we did :) (_Improved ...._)

```suggestion
- Improved handling of `networkIdle` lifecycle events. ([#647](https://github.com/grafana/xk6-browser/pull/647), [#644](https://github.com/grafana/xk6-browser/pull/644))

  Before this, a test script would time out when xk6-browser navigates to a website and waits for the `networkIdle` lifecycle event. This shouldn't change any existing behavior, but if you notice anything that doesn't work as expected, please let us know. 
```

Reply:
This was the refactoring, the `networkidle` fix was done in the previous release. I'll work with your suggestion 👍